src/Config/ObjectModel/RuntimeConfig.cs

```diff
@@ -191,13 +191,29 @@ public RuntimeConfig(
         this.Entities = Entities;
         this.DefaultDataSourceName = Guid.NewGuid().ToString();
 
+        if (this.DataSource is null)
+        {
+            throw new DataApiBuilderException(
+                message: "DataSource is a mandatory property in DAB Config",
+                statusCode: HttpStatusCode.UnprocessableEntity,
```

Comment:
I think that the message needs to be a bit more specific to show that the datasource is mandatory in all config files
```suggestion
                message: "DataSource is a mandatory property in all DAB configuration files",
```

Reply:
Since we are validating config file passed in the parameter `c {configfilename}`. This shouldnt be needed